database/user.go

```diff
@@ -130,12 +135,24 @@ type (
 	// TierLimits defines the speed limits imposed on the user based on their
 	// tier.
 	TierLimits struct {
-		UploadBandwidth   int   `json:"upload"`        // bytes per second
-		DownloadBandwidth int   `json:"download"`      // bytes per second
-		MaxUploadSize     int64 `json:"maxUploadSize"` // the max size of a single upload in bytes
-		MaxNumberUploads  int   `json:"-"`
-		RegistryDelay     int   `json:"registry"` // ms delay
-		Storage           int64 `json:"-"`
+		TierName          string `json:"tierName"`
+		UploadBandwidth   int    `json:"upload"`        // bytes per second
+		DownloadBandwidth int    `json:"download"`      // bytes per second
+		MaxUploadSize     int64  `json:"maxUploadSize"` // the max size of a single upload in bytes
+		MaxNumberUploads  int    `json:"-"`
+		RegistryDelay     int    `json:"registry"` // ms delay
+		Storage           int64  `json:"-"`
+	}
+	// TierLimitsPublic is a DTO specifically designed to inform the public
+	// about the different limits of each account tier.
+	TierLimitsPublic struct {
```

Comment:
If this is just for the API I'd move it to the API's package in `handler.go`

Reply:
I moved it to handler.go but I'd like to keep the name different. It makes it clear what the purpose of the type is without reading the comments.